### PR TITLE
Updated Readme and adding model

### PR DESCRIPTION
**Pull request by toshal-a**

### CODE REVIEW

**Review by vedic-partap — COMMENTED** [review]
It's a good habit to add Model Checkpoint in the keras model so that best model ( e.g. best Validation accuracy ) gets saved.
